<launch>
 <node name="LD14" pkg="ldlidar_sl_ros" type="ldlidar_sl_ros_node" output="screen">
 <param name="product_name" value="LDLiDAR_LD14"/>
 <param name="topic_name" value="LiDAR/LD14"/>
 <param name="port_name" value ="/dev/ttyUSB0"/>
 <param name="frame_id" value="lidar_frame"/>
 </node>
</launch>
